File: Micro Project/React/Common Project/ems-app/src/model/Employee.js

import Department from "./Department";


export default class Employee {
    
    id;
    empName;
    empSalary;
    department;
    deptId;

    constructor() {
        this.id = 0;
        this.empName = '';
        this.empSalary = 0;
        this.department = new Department();
    }

    
    setId(id) {
        this.id = id;
    }

    setEmpName(empName) {
        this.empName = empName;
    }

    setEmpSalary(empSalary) {
        this.empSalary = empSalary;
    }

    setDeptId(deptId) {
        this.department.deptId = deptId
    }
}

File: SpringBoot/SB-EMS-Project2/src/main/java/com/ems/model/Employee.java
package com.ems.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "tbl_employee1")
public class Employee {
	@Id
	@Column(name = "eid")
	private int id;

	@Column(name = "ename")
	private String empName;

	@Column(name = "salary")
	private int empSalary;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getEmpName() {
		return empName;
	}

	public void setEmpName(String empName) {
		this.empName = empName;
	}

	public int getEmpSalary() {
		return empSalary;
	}

	public void setEmpSalary(int empSalary) {
		this.empSalary = empSalary;
	}

	@Override
	public String toString() {
		return "Employee [id=" + id + ", empName=" + empName + ", empSalary=" + empSalary + "]";
	}

}
